From fac339a5f430dad9a903f46188a15851d477cc56 Mon Sep 17 00:00:00 2001
From: Pankaj Chauhan <pankaj.chauhan@freescale.com>
Date: Tue, 23 Dec 2014 17:27:29 +0530
Subject: [PATCH 147/245] devcore: Add msi_list to device

Add msi_list (head of MSI descriptor list) to struct device.
This is required to support non PCI MSI devices.

Signed-off-by: Pankaj Chauhan <pankaj.chauhan@freescale.com>
Change-Id: I5c7fe575a2245904a244d81adb1ba9a4137a5160
Reviewed-on: http://git.am.freescale.net:8181/33322
Reviewed-by: Stuart Yoder <stuart.yoder@freescale.com>
Tested-by: Stuart Yoder <stuart.yoder@freescale.com>
(cherry picked from commit db95305e1a7fa47ce0f7629afea951dbec81bfb6)
---
 include/linux/device.h | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/include/linux/device.h b/include/linux/device.h
index e7f49af..40f9cf1 100644
--- a/include/linux/device.h
+++ b/include/linux/device.h
@@ -808,6 +808,8 @@ struct device {
 
 	bool			offline_disabled:1;
 	bool			offline:1;
+
+	struct list_head msi_list;
 };
 
 static inline struct device *kobj_to_dev(struct kobject *kobj)
-- 
2.1.0.27.g96db324

